NARFO_BE/ControllerUnitTests/UnitTest1.cs

```diff
@@ -65,7 +65,7 @@ public async Task GetAllMembers_ShouldReturnAllMembers()
             var heroResponse = JsonConvert.DeserializeObject<Members>(jsonResponse);
             // Check if the Member is the same
             Assert.AreEqual(newMember.Id, heroResponse.Id);
-            Assert.AreEqual(newMember.FirstName, heroResponse.FirstName);
+            Assert.AreEqual(newMember.Firstname, heroResponse.FirstName);
```

Comment:
Why was this done?

Reply:
We deleted by mistake and added it again